748 navbar enhancement

foutoucour opened the pull request:
work on #748 

<!---
@huboard:{"order":5.548867186777082e-19}
-->

M4rtine committed:
navbar-prelayout

M4rtine committed:
new-icon-login

M4rtine committed:
[WIP]layout

M4rtine committed:
[WIP]layout

M4rtine committed:
[WIP]UX-design

M4rtine committed:
[749]external-font-UI

M4rtine committed:
[749]external-font-UI

M4rtine committed:
[748]pre-final-layout

M4rtine committed:
[WIP]navbar-right-menu

M4rtine commented:
ah mais je ne pensais pas que tu allais faire un pul request je n'ai pas fini cette branche.
Sinon j'aurais mis le pull request et le screenshot

foutoucour commented:
Une pull request peut etre faite pendant que le travail est en cours. Ca
permet de voir l'evolution du travail par exemple et de commenter le
travail avant que l'investissement est trop important.

Un peu comme quand tu montres ton layout avant de faire la compo :)

On Mon, Nov 28, 2016 at 9:03 AM David Mazeau <notifications@github.com>
wrote:

> ah mais je ne pensais pas que tu allais faire un pul request je n'ai pas
> fini cette branche.
> Sinon j'aurais mis le pull request et le screenshot
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cgstudiomap/cgstudiomap/pull/755#issuecomment-263278470>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhZ0kBUk-XiP9opOKthfTIp-QDG7Jgdks5rCt8cgaJpZM4K9Yc1>
> .
>
-- 
---
- Jordi Riera, Connecting Softwares and people.
Founder at www.cgstudiomap.org, Because everyone should be familiar with
the cg industry.

www.linkedin.com/in/jordirieracg/ <http://ca.linkedin.com/in/jordirieracg/>

M4rtine commented:
ouf je croyais avoir merdé encore

foutoucour commented:
nope :)

Et on merde pas, on evolue :)

On Mon, Nov 28, 2016 at 9:07 AM David Mazeau <notifications@github.com>
wrote:

> ouf je croyais avoir merdé encore
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cgstudiomap/cgstudiomap/pull/755#issuecomment-263279487>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAhZ0phoZQ3z0k1eHadDU5EEQ9PSmjcDks5rCuAxgaJpZM4K9Yc1>
> .
>
-- 
---
- Jordi Riera, Connecting Softwares and people.
Founder at www.cgstudiomap.org, Because everyone should be familiar with
the cg industry.

www.linkedin.com/in/jordirieracg/ <http://ca.linkedin.com/in/jordirieracg/>

M4rtine commented:
Est ce que tu as eu le temps de voir avec tout les trucs que tu as fais ?

M4rtine committed:
748-color-navbar-btn

M4rtine committed:
btn-connect

M4rtine committed:
navbar-colors-sizes

foutoucour commented:
Hey @M4rtine c'est quoi le problème que tu as avec le code?

Commit:
revision of the implementation of the search in navbar

Commit:
navbar: remove dependency to QueryURL

Commit:
remove unused file

foutoucour commented:
Salut,

J'ai retravaillé pour te debloquer. Le module de recherche dans la navbar devrait marcher maintenant.

Attention, il faut qu'on est un plan pour le cas des iframes ou on veut que les menus de la navbar ne soit pas visible, comme pour http://cmsms.motion-cafe.com/studios.
Comment on pourrait faire ca? Tu aurais une idée de design pour ces cas?
Le but serait de ne pas avoir différentes templates, plutôt des conditions. Je travaillerai les conditions une fois que le design sera fait.

M4rtine committed:
fix-dev-layers

M4rtine committed:
748-navbar-def

M4rtine committed:
[WIP]suppr-old-navbar

Commit:
Merge branch 'develop' into 748-navbar-enhancement

To sync website_iframe_host

Commit:
Merge branch 'develop' into 748-navbar-enhancement

Fix of the test of frontend_studio

Commit:
website_iframe_host: add navbar-off.css file

Implement a css that is only active under a specific condition.
That will help to design the navbar.

foutoucour commented:
Salut @M4rtine ,

J'ai implémenté la css qui permettra de designer la navbar pour les iframes qui veulent cacher la navbar.
voir http://cmsms.motion-cafe.com/studios par exemple.

La css est dans `/website_iframe_host/static/src/css/navbar-off.css`
Elle est chargée uniquement dans le cas ou le `website.iframe.host` a hide_navbar a True.

Pour tester, tu devras mettras ton localhost a hide_nabar.
Pour ca

1. `Dashboard > Settings > Website Iframe Hosts`
2. Édites `Localhost:8069`
3. Check la case `Hide navbar?`
4. Save

Normalement en rechargeant la page directory (ca ne marche pas sur les autres pages pour le moment), la navbar devrait avoir un background rouge, bien dégueux :)

Dis moi si ca marche de ton bord :)

M4rtine committed:
clean-old-search-navbar

M4rtine committed:
map-move-top

M4rtine commented:
@foutoucour J'ai réussi à faire un truc en attendant que je répare mon ordi. C'est pas super mais ça fera le job pour l'update. Dis moi si c'est bon comme ça

Le screenshot ne montre pas la nouvelle navbar (zone rougé) mais ça montre la suppression de l'ancienne barre de recherche

![navbar0001](https://cloud.githubusercontent.com/assets/6338568/21091638/e37860c6-c047-11e6-8656-af205ffd7f0a.jpg)

M4rtine commented:
@foutoucour Plus vite la version est en ligne plus vite je peux corriger en utilisant les simulateurs de FireFox en live ;)

Commit:
frontend_base: force company_status to be open by default.

Commit:
Merge branch 'develop' into 748-navbar-enhancement

Commit:
add selenium tests

Still experimentals, but informatives.

They are not ran by travis yet

Commit:
Cleaned up selenium tests

Commit:
website_iframe_host: more details in the logs

Commit:
frontend_studio: remove the search zone from the studio page

Commit:
website_iframe_host: rework the css that hide elements in navbar
